### PR TITLE
SSE Beta Docs

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: docs/sdk/server-side-sdks/go/go-gettingstarted.md [hunk 1]
```diff
@@ -52,13 +52,13 @@ initialize for any reason- it'll return as an error here.
...
-process in a separate go routine. When the channel receives a message, you will know the initialization process is complete.
+Additionally, local bucketing mode supports an optional `ClientEventHandler` parameter which will tell the sdk to run the initialization
+process in a separate go routine. This can be useful if you want to wait for the client to be fully initialized before proceeding.
 
 ```go
-onInitializedChannel := make(chan bool)
+onInitializedChannel := make(chan api.ClientEvent)
 options := devcycle.Options{
-    OnInitializedChannel:    onInitializedChannel,
+    ClientEventHandler:    onInitializedChannel,
```
Comment (ajwootto) [inline docs/sdk/server-side-sdks/go/go-gettingstarted.md:61]: isn't this a breaking change?

Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:61]: This was part of the major vbump we did already

File: docs/sdk/server-side-sdks/go/go-usage.md [hunk 2]
```diff
@@ -144,3 +144,23 @@ making bucketing decisions.
...
+options := devcycle.Options{
+    EnableBetaRealtimeUpdates:    true,
+    // other options omitted for this example
+}
+
+devcycleClient, err := devcycle.NewClient(sdkKey, &options)
+```
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long lived HTTP connection to receive updates on when there is a new config available. This reduces outbound network
```
Comment (ajwootto) [inline docs/sdk/server-side-sdks/go/go-usage.md:165]:
```suggestion
use a long-lived HTTP connection to receive updates when there is a new config available. This reduces outbound network
```

File: docs/sdk/server-side-sdks/go/go-usage.md [hunk 3]
```diff
@@ -144,3 +144,23 @@ making bucketing decisions.
...
 That data would be used in combination with EdgeDB data to make correct bucketing decisions before returning the
 variable's value.
+
+
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+To enable Beta Realtime Updates, pass in the `EnableBetaRealtimeUpdates` option to the SDK initialization:
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/go/go-usage.md:154]: Can you swap the order here, explain what the feature is, then show how to enable it.

File: docs/sdk/server-side-sdks/go/go-usage.md [hunk 4]
```diff
@@ -144,3 +144,23 @@ making bucketing decisions.
...
+```go
+options := devcycle.Options{
+    EnableBetaRealtimeUpdates:    true,
+    // other options omitted for this example
+}
+
+devcycleClient, err := devcycle.NewClient(sdkKey, &options)
+```
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/go/go-usage.md:164]: Also I think its ok to say we use Server Sent Events (SSE), some people know what that is.

File: docs/sdk/server-side-sdks/node/node-usage.md [hunk 5]
```diff
@@ -142,6 +142,25 @@ This will send a request to our EdgeDB API to save the custom data under the use
 
 In the example, Email and Country are associated to the user `test_user`. In your next variable call for the same `user_id`, you may omit any of the data you've sent already as it will be pulled from the EdgeDB storage when segmenting to experiments and features.
 
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+To enable Beta Realtime Updates, pass in the `enableBetaRealtimeUpdates` option to the SDK initialization:
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/node/node-usage.md:150]: same changes here.

File: docs/sdk/server-side-sdks/go/go-usage.md (hunk 3 above)
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-usage.md:154]: The order of the warning and all that? or just what the feature is portion?

File: docs/sdk/server-side-sdks/go/go-usage.md (hunk 4 above)
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-usage.md:164]: But you didn't want us using that language in the sdk option? I find that thought process a bit contradictory/confusing

**Review by jonathannorris — APPROVED** [review]

**Pull request merged**